OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -667,7 +668,23 @@ private void loadData(@NonNull CourseComponent courseComponent) {
         } else {
             // Remove bulk video download if the course has NO downloadable videos
             UiUtil.removeFragmentByTag(CourseOutlineFragment.this, "bulk_download");
-            errorNotification.showError(isVideoMode ? R.string.no_videos_text : R.string.no_chapter_text, null, -1, null);
```

Comment:
Here you can see we are handling the video subsection separately so that's why I implemented this